draft-ietf-trans-rfc6962-bis.md

```diff
@@ -1402,6 +1402,45 @@ Note that it is possible for a certificate to be logged more than once. If that
 is the case, the log MAY return more than one entry index. If the certificate is
 present in the log, then the log MUST return at least one entry index.
 
+## Retrieve Signed Tree Heads between Two Times
+
+GET https://\<log server>/ct/v2/get-sths
```

Comment:
Yeah I was about to say, the preface for 'optional messages' says they are 'not required for correct operation of logs or their clients', but you might make the case that get-sths is pretty important for correct operation of an auditing monitor...